cypress/e2e/contactForm.cy.js

```diff
@@ -1,7 +1,9 @@
-import ContactFormPageObject from '../support/pages/contactForm.pageObject';
+import ContactFormPageObject
```

Comment:
The import of 'ContactFormPageObject' is split into two lines. This can make the code less readable. It would be better to keep it on one line unless it exceeds the maximum line length.